### ISSUE TITLE
[FR] Include Dockerfile at Analyzer Project level

### ISSUE THREAD

**Issue by milesflo** [TheHive-Project/Cortex-Analyzers#606]
**Feature description**
Include Dockerfile at project level to allow the creation of local Dockerized analyzers

**Describe the solution you'd like**
In each Analyzer directory, include a `Dockerfile` specifying the necessary build steps & entry point, with metadata labels.

**Additional context**
Doing this will be a great step in the direction [this blog post](https://blog.thehive-project.org/2019/11/20/unleash-the-power-of-dockerized-analyzers-in-5-minutes/) was suggesting: moving from **process** based analyzer execution to **docker** based.

It will also address a concern myself and other have expressed by the project's use of docker helper scripts rather than the native Docker deployment system.

**Comment by nadouani** [comment]
Hey @milesflo 

I'm back to this issue + pull request. I think that the original requirement makes sense: "Allow the creation of local dockerized analyzers" but there are two issues here:
- You have created the Github issue 17 days ago
- You have started working on it right away without any discussion/validation with the project team
- You have missed the forllowing repository that already do the job automatically through our CI pipeline: https://github.com/TheHive-Project/cortex-worker-builder

Let me explain it: Our CI pipeline (a private drone server) builds the docker images automatically (in addition to all the other packages), publish them etc... The docker images it produces contain all the metadata you have added in your script (we use the official docker metadata properties BTW).

We decided to not store the docker files in every analyzer folder.

That being said, we can update the script https://github.com/TheHive-Project/cortex-worker-builder to allow persisting dockerfiles for selected analyzers if you need to build them locally.

**Comment by milesflo** [comment]
I disagree with your decision, and would appreciate if you would update that script.

**Comment by Xyrodileas** [comment]
I support Milesflow's request. I've worked to redo the setup with Drone and was not able to. Really diminish the ability to contribute, test new features easily and trust the project for the long run as I have to trust binary hosted on a 3nd party website.
This is a major concern when wanting to migrate to TheHive/Drone, contribute and ditch the previous ticket management system.
